samples/cpp/rag/text_embeddings.cpp

```diff
@@ -17,8 +17,8 @@ int main(int argc, char* argv[]) try {
 
     ov::genai::TextEmbeddingPipeline pipeline(models_path, device, config);
 
-    ov::genai::EmbeddingResults embeddings = pipeline.embed_documents(documents);
-
+    ov::genai::EmbeddingResults documents_embeddings = pipeline.embed_documents(documents);
+    ov::genai::EmbeddingResult query_embedding = pipeline.embed_query("What is the capital of France?");
```

Comment:
shall it be a separate argument in the sample?

Reply:
I think it's ok to not have it as an argument to simplify run command and to encourage for sample modification.